### ISSUE TITLE
Keyboard Bindings (DEL, CTRL-V,...) not working after using Zoom-Slider in Toolbar [KitchenSinkAngular6]

### ISSUE THREAD

**Issue by MichaelStroehlein** [clientIO/joint#1203]
After using the zoom slider in the toolbar to increase or decrease the zoom level, the registered keyboard keybindings are not working any more. 
There's no error in the console or anything - the listeners for the keybindings  in the `KeyboardService `are not called.
This issue is observed in KitchenSinkAngular6 Demo shipped with Rappid 3.0.1 and also occurs in our project.

**Comment by kumilingus** [comment]
Hi Michael,
thanks for letting us know. We have fixed the issue and we will ship it with the next release. For the time being please replace: 

```js
new joint.mvc.View({ el: this.el });
```

with: 

```js
const view = new joint.mvc.View({ el: this.el });
view.delegateEvents({
  'mouseup input[type="range"]': (evt) => evt.target.blur()
});
```

It's important to remove the focus from the input range after the user interaction.

**Comment by MichaelStroehlein** [comment]
Hello kumilingus,
thank you very much for providing a workaround - it's working in our project as well.

**Comment by kumilingus** [comment]
Fixed in Rappid `v3.1.1`.
